be/src/exprs/agg/aggregate_factory.cpp

```diff
@@ -854,6 +886,23 @@ AggregateFuncResolver::AggregateFuncResolver() {
 
     add_aggregate_mapping<TYPE_BIGINT, TYPE_BIGINT, true>("count");
 
+    ADD_ALL_TYPE1("max_by", TYPE_BOOLEAN);
```

Comment:
why `ADD_ALL_TYPE1`, rather than `ADD_ALL_TYPE`?

Reply:
because column1 and column2 can be different types.  that means it can be 256 combinations